src/azure-cli/service_name.json

```diff
@@ -421,18 +416,18 @@
   },
   {
     "Command": "az role",
-    "AzureServiceName": "Azure role-based access control (RBAC)",
-    "URL": "https://docs.microsoft.com/azure/role-based-access-control/overview"
+    "AzureServiceName": "Security",
```

Comment:
For consistency with Azure PowerShell and ease to find, I would recommend "Resources"

Reply:
Security is too vague for a TOC node in my opinion. Besides the alignment with Azure PowerShell, the primary reason why RBAC should be under 'Resources' is because it is built on Azure Resource Manager and manages access to Azure Resources, it is about "Resource management".

`az ssh` and `az sshkey` could be under an ssh node.

Resource Graph commands should be under "Resource Graph".